packages/vscode-extension/src/treeview/treeViewManager.ts

```diff
@@ -64,6 +65,24 @@ class TreeViewManager {
       );
       developmentTreeviewProvider.refresh();
     }
+    if (TreatmentVariableValue.previewTreeViewCommand) {
```

Comment:
Does this command show for both new app and existing app?

Reply:
Fixed comment.